pkgs/by-name/un/unifi/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  dpkg,
+  fetchurl,
+  nixosTests,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "unifi-controller";
+  version = "8.6.9";
+
+  # see https://community.ui.com/releases / https://www.ui.com/download/unifi
+  src = fetchurl {
+    url = "https://dl.ui.com/unifi/${version}/unifi_sysvinit_all.deb";
+    sha256 = "sha256-004ZJEoj23FyFEBznqrpPzQ9E6DYpD7gBxa3ewSunIo=";
+  };
+
+  nativeBuildInputs = [ dpkg ];
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg-deb -x $src ./
+    runHook postUnpack
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cd ./usr/lib/unifi
+    cp -ar dl lib webapps $out
+
+    runHook postInstall
+  '';
+
+  passthru.tests = {
+    unifi = nixosTests.unifi;
+  };
+
+  meta = with lib; {
+    homepage = "http://www.ubnt.com/";
+    description = "Controller for Ubiquiti UniFi access points";
+    sourceProvenance = with sourceTypes; [ binaryBytecode ];
+    license = licenses.unfree;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [
+      globin
+      patryk27
+    ];
```

Comment:
`with lib;` is the more common pattern and since both snippets do the same thing, I don't think it's worth changing it here - you'll say `don't use with lib;`, in two weeks someone will say `use with lib;` etc.; unless there's a specific style recommendation, I'd keep whatever's there already.